package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "truex-shared",
-  "version": "1.6.13",
+  "version": "1.7.1",
```

Comment:
I haven't made the 1.7.1 tag yet, figure I'd wait a bit given the current state

Reply:
Good call! And thanks for handling the bump up to `1.7`. We've been meaning to do that for a bit